src/composites/ConfigurationContainer/ImportanceAdjustment.tsx

```diff
@@ -0,0 +1,114 @@
+import { useAtom, useAtomValue } from "jotai";
+import React, { ChangeEvent, useState } from 'react';
+import { State } from "../../state";
+import { Button, Flex, Text, HoverCard, Link, Strong, Callout, Box, Separator, IconButton, Select } from "@radix-ui/themes";
+import { InfoCircledIcon, GearIcon, Cross2Icon, Cross1Icon } from "@radix-ui/react-icons";
+import * as Dialog from '@radix-ui/react-dialog';
+import "../Style/Dialog.css"
+import { ProfileSelection } from "./ImportanceAdjustment/ProfileSelection";
+import { AdjustmentTable } from "./ImportanceAdjustment/AdjsutmentTable";
+import { Profile } from "../../types";
+
+
+export const ImportanceAdjustment = () => {
+    const [selectedProfile, setSelectedProfile] = useState<Profile | Profile[] | null>(null);
+
+    // haddle the profile selection and apply
+    const [isProfileApplied, setIsProfileApplied] = useState(false);
+
+    const handleProfileApply = (profile: Profile[] | null) => {
+        setSelectedProfile(profile);
+        setIsProfileApplied(true);
```

Comment:
You can use `const isProfileApplied = profile === null` to achieve the same effect. When you want to reset the profile, set the selected profile to `null`. This will avoid issues of `selectedProfile` and `isProfileApplied` going out of sync.

Reply:
addressed